environs/cloudspec/cloudspec.go

```diff
@@ -64,6 +69,26 @@ func (cs CloudSpec) Validate() error {
 	return nil
 }
 
+// IsLocalhostCloud returns true if the cloud is an LXD localhost cloud.
+func (cs *CloudSpec) IsLocalhostCloud() (bool, error) {
+	if cs.Type != lxdnames.ProviderType &&
+		cs.Type != lxdnames.DefaultCloud {
```

Comment:
There's a helper for this - `lxdnames.IsDefaultCloud()`
But regardless, I'm not sure we want it. So long as it's a lxd cloud and the endpoint address is local, we don't care what it's called.

Reply:
done